File: Java/validAnagram.java
public class Solution {
    public boolean isAnagram(String s, String t) {
        String newS = generateSortedString(s);
        
        String newT = generateSortedString(t);
        
        if(newT.equals(newS))
            return true;
        else
            return false;
        
    }
    
    String generateSortedString(String s) {
        char[] chars = s.toCharArray();
        Arrays.sort(chars);
        return new String(chars);
    }
}

File: C#/validAnagram.cs
public class Solution {
    public bool IsAnagram(string s, string t) {
        return orderString(s).Equals(orderString(t));
        
    }
    
    public string orderString(string str) {
        char[] characters = str.ToArray();
        Array.Sort(characters);
        return new string(characters);
    }
}